docs/contributing-to-airbyte/how-to-contribute-pr.md

```diff
@@ -0,0 +1,112 @@
+
+1. A **Major Version** increase 
+2. An Airbyte Engineer to follow the  [Connector Breaking Change Release Playbook](https://docs.google.com/document/u/0/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit) before merging.
+
+
+### Types of Breaking Change(s):
+A breaking change is any change that will require users to take action before they can continue to sync data. The following are examples of breaking changes:
+
+- **Spec Change** - The configuration required by users of this connector have been changed and syncs will fail until users reconfigure or re-authenticate.  This change is not possible via a Config Migration 
+- **Schema Change** - The type of a property previously present within a record has changed
+- **Stream or Property Removal** - Data that was previously being synced is no longer going to be synced.
+- **Destination Format / Normalization Change** - The way the destination writes the final data or how normalization cleans that data is changing in a way that requires a full-refresh.**
+- **State Changes** - The format of the source’s state has changed, and the full dataset will need to be re-synced
+
+
+### Checklist for Contributors
+
+First, ask yourself does your change correspond to any of the breaking changes above?
+
+If so then follow this checklist below
+
+- [ ] Apply the label breaking-change to your PR
+- [ ] Apply a Major Version bump to your PR. See [Semantic Versioning for Connectors](#semantic-versioning-for-connectors) for more information.
+- [ ] Prepend your PR title with the 🚨🚨 emoji.
+- [ ] Add a section to the PR description titled Breaking Change that describes why this is a breaking change, and if possible how you can migrate users and/or rollback
+- [ ] Assign an Airbyte Engineer through the `airbytehq/connector-operations` group and have them start the [Connector Breaking Change Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit#)
+
+
+### Steps to contributing code
+
+#### 1. Open an issue, or find a similar one.
+Before jumping into the code please first:
+1. Verify if an existing [connector](https://github.com/airbytehq/airbyte/issues) or [platform](https://github.com/airbytehq/airbyte-platform/issues) GitHub issue matches your contribution project.
+2. If you don't find an existing issue, create a new [connector](https://github.com/airbytehq/airbyte/issues/new/choose) or [platform](https://github.com/airbytehq/airbyte-platform/issues/new/choose) issue to explain what you want to achieve.
+3. Assign the issue to yourself and add a comment to tell that you want to work on this.
+
+This will enable our team to make sure your contribution does not overlap with existing works and will comply with the design orientation we are currently heading the product toward.
+If you do not receive an update on the issue from our team, please ping us on [Slack](https://slack.airbyte.io)!
```

Comment:
If you do not receive an update on the issue from our team, please ping us on #contributions channel on [Slack](https://slack.airbyte.io)!